### PR TITLE
Cleaning tmp merge

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

**Comment by mvidner** [comment]
LGTM

File: src/include/bootloader/routines/lib_iface.rb [hunk 1]
```diff
@@ -189,46 +190,41 @@ def SetGlobal(globals)
...
     # Set the device mapping (Linux <-> Firmware)
     # @param [Hash{String => String}] device_map a map from Linux device to Firmware device identification
     # @return [Boolean] true on success
     def SetDeviceMap(device_map)
-      arg_data = TmpYAMLFile.new(device_map)
-
-      Builtins.y2milestone("Storing device map")
-      run_pbl_yaml "SetDeviceMapping(#{arg_data.path})"
+      Builtins.y2milestone("Storing device map #{device_map}")
+      TmpYAMLFile.open(device_map) do |args_data|
```
Comment (mvidner) [inline src/include/bootloader/routines/lib_iface.rb]: `arg_data`

Comment (jreidinger) [inline src/include/bootloader/routines/lib_iface.rb]: thanks for catch, fixed.

**Comment by mvidner** [comment]
LGTM, thanks.

**Pull request merged**